01who_we_are/02values.md

```diff
@@ -7,31 +7,72 @@ We take our core values very seriously. They're the underpinning for what we do,
+
+Some examples of this value in action are:
+
+- Our regular process of peer review for our code and content, where we provide feedback and encourage for improvement.
+- Contributing our standard high quality videos for free to enhance the Drupal community documentation.
 
 **Feed creativity**
 
-Creativity creates space for innovative ideas and inventive solutions. We embrace the full creative process, which includes failing, learning, and trying again.
+Creativity creates space for innovative ideas and inventive solutions. We embrace the full creative process, which includes failing, learning, and trying again, while providing a safe and respectful environment. We are open to the whole breadth of creative change, and are not afraid to change existing things as well as creating something new. Through openly sharing our new ideas in the wider world we aim to also foster creativity in others.
+
+Some examples of this value in action are:
+
+- We have new ideas for how to deliver our content, from written tutorials to new hands-on project guides. Each time it started with a new idea, often based on customer feedback, putting the idea out there, and then continuing to iterate to fix failings and make improvements.
+- We re-evaluated our entire business strategy and ended up expanding our business model and changing the company name. 
+
 
 **Spread happiness**
 
-We believe in the power of happiness. We encourage playfulness, positivity, and kindness in everything we do.
+We believe in the power of happiness. We encourage playfulness, positivity, and kindness in everything we do. We try not to take ourselves too seriously and recognize that being approachable and kind makes learning more fun and less stressful. We are also aware of the impact our attitude has on the people around us and that impact spreads far beyond our personal and professional interactions.
+
+Some examples of this value in action are:
+
+- Creating fun examples in our learning materials, like the ice cream shop example for learning plugins or the batman cartoon for events.
+- Fun stickers! Our sparkly Drupal ponies continue to be a favorite even today.
+
 
 **Work together**
 
-We keep open minds and listen to each other to find common ground and consensus. We support and help each other in ways that make our entire team stronger.
+We believe in Open Source, not only in open software but also in the principles of collaboration and sharing. We fully engage with Open Source communities through contribution and have incorporated these fundamentals throughout our company. We keep open minds and listen to each other to find common ground and improve each other’s work without judgment. We apply these same principles when working with our customers to solve their problems.
+
+Some examples of this value in action are:
+
+- Sharing our Employee Handbook publicly
+- Collaborating on the Drupal 8 User Guide and sharing our enhancements (adding videos)
+
```

Comment:
Does it make sense to mention our efforts towards helping the Node.js website redesign effort here, or would that still be a bit premature?

Reply:
Yeah, I think it is a little early since most of the world cant actually see the work or its results.